src/app/portlets/dot-edit-page/content/dot-edit-content.component.ts

```diff
@@ -464,6 +464,9 @@ export class DotEditContentComponent implements OnInit, OnDestroy, OnSaveDeactiv
                 if (this.originalValue) {
                     this.ngZone.run(() => {
                         this.isModelUpdated = !_.isEqual(model, this.originalValue);
+                        if (this.isModelUpdated) {
+                            this.saveContent();
```

Comment:
Did you tests what happen if I move things around faster than the save request is done? You can fake that with a delay.